apps/dashboard/config/configuration_singleton.rb

```diff
@@ -36,6 +36,10 @@ def ood_version
     @ood_version ||= (ood_version_from_env || version_from_file('/opt/ood') || version_from_git('/opt/ood') || "Unknown").strip
   end
 
+  def ood_bc_ssh_to_compute_node
+    ENV['OOD_BC_SSH_TO_COMPUTE_NODE'] || true
```

Comment:
Sorry I realized the logic is slightly more complicated: we want to return true if `ENV['OOD_BC_SSH_TO_COMPUTE_NODE']` is `nil` otherwise we want to return the result of the environment variable after it has been run through the `o_bool` method.